contributing/documentation/overview.rst

```diff
@@ -249,21 +249,21 @@ GitHub, click on the **Show all checks** link and finally, click on the
 Build the Documentation Locally
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you have Docker installed on your machine, run these commands to build the
-docs:
+If you have Docker installed on your machine, this command will build
+the image, compile the docs, and start to watch for changes and
+automatically compile them:
 
 .. code-block:: terminal
 
-    # build the image...
-    $ docker build . -t symfony-docs
+    # We use the same user as outside to avoid permissions issues with generated files
+    $ CURRENT_UID=$(id -u):$(id -g) docker-compose up
 
-    # ...and start the local web server
-    # (if it's already in use, change the '8080' port by any other port)
-    $ docker run --rm -p 8080:80 symfony-docs
-
-You can now read the docs at ``http://127.0.0.1:8080`` (if you use a virtual
+You can now read the docs at ``http://localhost:8080`` (if you use a virtual
 machine, browse its IP instead of localhost; e.g. ``http://192.168.99.100:8080``).
 
+If por `8080` is already in use, you can change the mapping on `docker-compose.yaml`
```

Comment:
`por` -> `port`

Also, all single quoted code should be double quoted:

```rst
`8080` -> ``8080``, etc.
```

Reply:
Good catch!
I am just too used to MD and need to switch my mind when using rst.